dbt_dagster_project/dbt_dagster_project/assets.py

```diff
@@ -0,0 +1,44 @@
+import json
+from typing import Any, Mapping
+
+from dagster import (AssetExecutionContext, DailyPartitionsDefinition,
+                     OpExecutionContext)
+from dagster_dbt import (DagsterDbtTranslator, DbtCliResource, dbt_assets,
+                         default_metadata_from_dbt_resource_props)
+
+from .constants import dbt_manifest_path
+
+
+@dbt_assets(manifest=dbt_manifest_path, exclude="fct_reviews")
+def dbtlearn_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
+    yield from dbt.cli(["build"], context=context).stream()
+
+
+# many dbt assets use an incremental approach to avoid
+# re-processing all data on each run
+# this approach can be modelled in dagster using partitions
+daily_partitions = DailyPartitionsDefinition(start_date="2022-01-24")
+
+class CustomDagsterDbtTranslator(DagsterDbtTranslator):
+    def get_metadata(self, dbt_resource_props: Mapping[str, Any]) -> Mapping[str, Any]:
+        metadata = {"partition_expr": "date"}
+        default_metadata = default_metadata_from_dbt_resource_props(dbt_resource_props)
+        return {**default_metadata, **metadata}
```

Comment:
What is this function doing? 

In our implementation, we use https://docs.dagster.io/_apidocs/libraries/dagster-dbt#dagster_dbt.DagsterDbtTranslator.get_group_name 
I find it personally very nice to teach what a custom translator is doing

Reply:
I'm happy you like most of the code; it's @geoheil 's after all ;)